pkg/deb/usr/lib/systemd/system/peerbanhelper.service

```diff
@@ -1,11 +1,13 @@
 [Unit]
 Description=PeerBanHelper Service
 After=network.target
+
 [Service]
 User=peerbanhelper
 WorkingDirectory=/etc/peerbanhelper
-ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
+ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Dpbh.log.level=WARN -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
```

Comment:
According to the FHS specification, data files should not be stored in the `/etc` directory, I'll ask @Ghost-chu  to add an option `pbh.configdir`, let's put the data files under `/var/lib` and leave config in `/etc`

Reply:
@Gaojianli Changes applied, also I think it's un-necessary to package config.yml or profile.yml in .deb, because PeerBanHelper will auto-generate them.

Reply:
The config files need to be inside the DEB otherwise `conffiles` can't be used.